src/scss/components/_carousel.scss

```diff
@@ -2,8 +2,11 @@
   --time-transitions: 0.3s;
 }
 
-.general-container {
+.carousel-general {
   @include grid;
+
+  position: relative;
+  height: 0;
```

Comment:
These two properties are not necessary.

Reply:
If you keep them, it will break the layout of the page

Reply:
Ok, understood